ims/common/bmi_config.py

```diff
@@ -36,8 +40,12 @@ def parse_config(cfg):
     cfg.option(constants.DRIVER_SECTION, constants.ISCSI_DRIVER_OPT)
     cfg.option(constants.DRIVER_SECTION, constants.FS_DRIVER_OPT)
 
-    # Mandatory Sections (Typically Driver Sections)
+    # ISCSI section
     cfg.section(constants.ISCSI_SECTION)
+    cfg.option(constants.ISCSI_SECTION, constants.PRIMARY_ISCSI_HOST)
+    cfg.option(constants.ISCSI_SECTION, constants.SECONDARY_ISCSI_HOST)
```

Comment:
Do we only need a primary and secondary host?

Reply:
Yep, I can't specify more than 2 paths anyway in the ipxe file.